sklearn/tree/_export.py

```diff
@@ -986,7 +1005,26 @@ def export_text(
     check_is_fitted(decision_tree)
     tree_ = decision_tree.tree_
     if is_classifier(decision_tree):
-        class_names = decision_tree.classes_
```

Comment:
Uhm it seems that I misunderstood something when reading the documentation at first. We already uses `decision_tree.classes_`. So we don't need `"numeric"` and any deprecation (which is a good news).

Sorry to have brought this way. We will need to modify (remove) the code :)